<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IcalDirsAgentSettingsWidget</class>
 <widget class="QWidget" name="IcalDirsAgentSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>386</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="ktabwidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Directory</string>
      </attribute>
      <layout class="QFormLayout" name="verticalLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Di&amp;rectory:</string>
         </property>
         <property name="buddy">
          <cstring>kcfg_Path</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KUrlRequester" name="kcfg_Path"/>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="pathLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Select the directory whose contents should be represented by this resource. If the directory does not exist, it will be created.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="kcfg_ReadOnly">
         <property name="text">
          <string>Read only</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="readOnlyLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="visible">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>When read-only mode is enabled, no changes will be written. Read-only mode is automatically enabled when the selected file does not support write access.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>4</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tuning</string>
      </attribute>
      <layout class="QFormLayout" name="verticalLayout_5">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="runingLabel">
         <property name="text">
          <string>The options on this page allow you to change parameters that balance data safety and consistency against performance. In general you should be careful with changing anything here, the defaults are good enough in most cases.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="autosaveLabel">
         <property name="text">
          <string>Autosave delay:</string>
         </property>
         <property name="buddy">
          <cstring>kcfg_Path</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KPluralHandlingSpinBox" name="kcfg_AutosaveInterval">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>138</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KPluralHandlingSpinBox</class>
   <extends>QSpinBox</extends>
   <header>kpluralhandlingspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_ReadOnly</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_AutosaveInterval</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_ReadOnly</sender>
   <signal>toggled(bool)</signal>
   <receiver>autosaveLabel</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
